Ghidra/Features/Decompiler/src/decompile/cpp/space.cc

```diff
@@ -268,7 +270,10 @@ uintb AddrSpace::read(const string &s,int4 &size) const
     }
   }
   catch(LowlevelError &err) {	// Name doesn't exist
-    offset = strtoul(s.c_str(),&tmpdata,0);
+    offset = strtoull(s.c_str(),&tmpdata,0);
+    if (offset == ULLONG_MAX) {
```

Comment:
would you want to also check errno for ERANGE to determine if you should throw?

Reply:
Done. From what I understand, I also need to check the `endptr` argument in case of parsing errors so I've added checks for that too.